rst_files/kalman.rst

```diff
@@ -101,71 +101,42 @@ This density :math:`p(x)` is shown below as a contour map, with the center of th
 .. code-block:: julia
   :class: collapse
 
-  using Plots, LaTeXStrings, LinearAlgebra
+  using Plots, LaTeXStrings, LinearAlgebra, Compat, Distributions
```

Comment:
We shouldn't be using LaTeXStrings (Compat should have already been loaded)

Reply:
Divide the `using` and `gr` statements from the other content on this cell.

Reply:
Took out `LaTeXStrings` and `Compat`. Same is true for `LinearAlgebra` as well.